presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -292,11 +292,11 @@ public void testDynamicFilter()
         assertEquals(stats.getReplicatedDynamicFilters(), 0);
         assertEquals(ImmutableSet.copyOf(stats.getDynamicFilterDomainStats()), ImmutableSet.of(
                 new DynamicFilterDomainStats(
-                        filterId1, getExpectedDomainString(1L, 2L), 2, 0),
+                        filterId1, getExpectedDomainString(1L, 2L), 2, 0, Optional.empty()),
                 new DynamicFilterDomainStats(
-                        filterId2, getExpectedDomainString(2L, 3L), 2, 0),
+                        filterId2, getExpectedDomainString(2L, 3L), 2, 0, Optional.empty()),
                 new DynamicFilterDomainStats(
-                        filterId3, Domain.none(INTEGER).toString(session.toConnectorSession()), 0, 0)));
+                        filterId3, Domain.none(INTEGER).toString(session.toConnectorSession()), 0, 0, Optional.empty())));
```

Comment:
fmt: all args to `new DynamicFilterDomainStats` should be on the `new DynamicFilterDomainStats` line, or each on separate line